pom.xml

```diff
@@ -189,6 +194,24 @@ limitations under the License.]]></inlineHeader>
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.jacoco</groupId>
+        <artifactId>jacoco-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <goals>
+              <goal>prepare-agent</goal>
+            </goals>
+          </execution>
+          <execution>
+            <id>report</id>
+            <phase>prepare-package</phase>
+            <goals>
+              <goal>report</goal>
```

Comment:
It would be nice to exclude the mapper generated classes such as `ProductHelper__MapperGenerated` as they are pulling the code coverage down.
You can also exclude test classes like CassandraEndpoint, CassandraNameConverterEndpoint, etc.
And finally, you can also exclude Quarkus generated classes like `CassandraClientBuildTimeConfig$$accessor`.
We can do this in a follow-up ticket.